app/components/request/configs.js

```diff
@@ -18,6 +18,12 @@ export default Component.extend(CanTriggerBuild, TriggerBuild, {
 
   preview: service('request-config'),
   features: service(),
+
+  status: 'closed',
+  closed: equal('status', 'closed'),
+  customizing: equal('status', 'customize'),
+  previewing: equal('status', 'preview'),
```

Comment:
We usually strive to extract dictionary values to constants and re-use them whenever possible, to avoid hardcoding and simplify potential refactoring, e.g.:
```js
export const STATUSES = {
  CLOSED: 'closed',
  CUSTOMIZE: 'customize',
  PREVIEW: 'preview'
};

export default Component.extend({
  status: STATUSES.CLOSED,
  closed: equal('status', STATUSES.CLOSED),
  customizing: equal('status', STATUSES.CUSTOMIZE),
  previewing: equal('status', STATUSES.PREVIEW),
})
```
How does it sound to you?

Reply:
ok, done. i've also then reused the constants in the code further down (in various `on*` handlers)

however, all this super-all-caps code now is a little hard on the eyes for my taste. i'll gladly keep it if this is the right JS or Ember convention.

as a bit of an alleviation ... would it be ok to use a singular `STATUS` constant and keep the keys lowercase? for example `STATUS.closed`? i usually keep such constants singular in my code because i only ever refer to one single status after all.

Reply:
@svenfuchs thank you, it looks great to me 👍 This is not a particular JS or Ember convention, just a common good practice that we use across the app. Using singular form for dictionary name is totally fine with me 👍 The names are usually totally up to developer. Although, for consistency I'd prefer keeping the keys uppercase as well. I think it's just a matter of preference and habit - with all-caps you can easily find/see constant and dictionary values even briefly scanning a file. Does it make sense to you?